.github/workflows/nestml-build.yml

```
@@ -52,7 +52,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        nest_branch: ["v2.20.2", "v3.0", "v3.1", "v3.2", "v3.3", "master"]
+        nest_branch: ["v2.20.2", "v3.0", "v3.1", "v3.2", "v3.3", "v3.4", "master"]
```

Shall we do a test run with the latest release candidate for now?
```suggestion
        nest_branch: ["v2.20.2", "v3.0", "v3.1", "v3.2", "v3.3", "v3.4_rc2", "master"]
```